### PR TITLE
Speed and Clarity

### PR DESCRIPTION
**Pull request by robee**
New rule to speed up the game and rule clarifications.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,10 +66,17 @@
...
->The player with the highest point value at this time wins the game.
 
 [[1](https://github.com/stolksdorf/nomic/pull/1)]
 
 ----
+
+----
+
+##### 108: Rule of Expediency
+> Proposed rules are automiatically accepted exactly 24 hours after the proposal of the rule if the rule has not already been rejected.
```
Comment (stolksdorf) [inline README.md]: automiatically -> automatically

**Comment by stolksdorf** [comment]
AGREED, 2/2.

I dig the clarifications.

File: README.md (hunk 1 above)
Comment (robee) [inline README.md]: i have speling problems.

**Comment by JMTyler** [comment]
(1) Fix the pull request identifier on Rule 108 (should be [3]) 
(2) Add the same identifier to any & all rules you modified, in the same fashion as presently depicted by Rule 106

Then, you will have my vote.

**Comment by robee** [comment]
@JMTyler Your requested changes have been made.

**Comment by JMTyler** [comment]
AGREED, 3/3

Beauty.

**Comment by lpellow** [comment]
Clarity on rule #108: "if the rule has not already been rejected." - does this mean at least one person has rejected? Or it simply did not achieve majority?

**Comment by ekmccann** [comment]
Similarly to LP I think this rule is somewhat unclear with majority voting. What will "rejection" mean for majority? Are you essentially just saying "votes are counted after 24h to determine if a rule passes"?

**Comment by mimosacb** [comment]
If Katie's interpretation is correct I will approve this change.

**Comment by DukeKataron** [comment]
AGREED 4/4

Sounds aight to me regardless of the clarity talk on #108. I assume it's just set up to be flexible with the idea of changing voting rules, so I'ma gonna agree to it before going to bed

Also does agreed need to be all in caps? 'cause that makes me feel like I'm vigorously agreeing with this stuff that I only want to agree with on a regular level. but Jared's, Scott's, and the example all used all caps so I figured I wouldn't rock the textual boat.

**Comment by robee** [comment]
@lpellow @McCsocks I have clarified rule 108.

**Comment by lpellow** [comment]
Very good, very good. 5/5.

**Comment by ekmccann** [comment]
Agh. Sorry for being the worst and/or an idiot, but now it reads to me as if voting is essentially unnecessary since all rules always pass. "Rules are accepted....regardless of whether a decision has been made." i.e. even if we vote to reject, the rule is accepted regardless of that.  

In the current form I can't accept it. The previous revision was actually more clear to me than this one. I do appreciate the reference to rule 104 re: what constitutes a decision (that fixes my issue with the previous revision re: what a rejection is).

IMO I would word it as "Proposed rules are automatically accepted exactly 24 hours after the proposal if a decision has not been reached as per rule 104." 

Sorry for being literally hitler.

**Comment by mimosacb** [comment]
It's reading to me like:

> all rules pass regardless of voting

I'd change it to say:

> Exactly 24 hours after the most recent rule-change the polls are closed and the vote is tallied as per rule 104. Players who fail to vote before the the polls close are not included to calculate majority for that turn. In the event that no votes are cast the rule-change is passed.

**Comment by robee** [comment]
@mimosacb Would you accept this

Exactly 24 hours after a proposal is created votes can no longer be cast and the decision criteria are evaluated (ie. vote is tallied) as per rule 104. Players who fail to vote before this time are not included in determining whether a decision is made as per rule 104. In the event that no votes are cast the rule-change is passed.

My edits to your version are trying to have more precise language (removing vernacular) and allowing the wording to work with any version of rule 104.

This is a slight deviation from my current intention with the rule but I think it is more fair and it does reward the people who are participating.

**Comment by mimosacb** [comment]
Works for me, I'd approve it.

**Comment by ekmccann** [comment]
I would also approve that wording.

**Comment by stolksdorf** [comment]
Too many changes, and process taking too long. Switching my vote to DECLINE.

**Pull request closed without merging**